pkg/test/generic/generic.go

```diff
@@ -42,26 +40,3 @@ func WithAnnotationsPopulated() Option {
 		}
 	}
 }
-
-// ChecksumFunc defines a function signature for checksumming objects.
-type ChecksumFunc func(runtime.Object) string
```

Comment:
The `ChecksumOfObjectFunc` and `ChecksumOfObjectsFunc` in `pkg/controller/utils/utils.go` are also not used by anything any more.

Also, while in that file, `GetChecksumOfObjects` can call `GetChecksumOfObject` to produce the same result without duplicating code.
```
func GetChecksumOfObjects(objects ...interface{}) (string, error) {
  return GetCheckumOfObject(objects)
}
```